### DISCUSSION TITLE
Vivo Apps apks can't be uninstalled or disabled in latest UAD

### DISCUSSION THREAD

**Discussion by SM-03** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#693] [Q&A] [answered]
Hello, 
I recently tried debloating a Vivo phone running on Funtouch OS based on Android 14. Most of the Vivo app packages cannot be uninstalled or disabled from the latest UAD v1.1.0, showing a failure pop-up screen (I forgot to take a screenshot). 

Is there any way to uninstall or disable these vivo apppackes? 
Anyone has any prior experience with vivo/funtouchOS, pls let me know

I am sharing the list here & requesting the developer to add these packages for uninstallation in the next version.

iManager:  com.vivo.imanager
Jovi (Vivo Assistant): com.vivo.assistant, com.vivo.assistantfunction, com.vivo.hiboard
Theme: com.bbk.theme, com.bbk.theme.resources
Tips: com.vivo.tips
Vtrust: com.vivo.customtool
Vivo cloud: com.bbk.cloud
Vivo Store: com.vivo.website
V-AppStore (Hot Apps Hot Games): com.vivo.appstore

**Comment by AnonymousWP** [comment]
Please post logs.

**Reply by SM-03** [reply]
> Please post logs.

where to find the log?  I'll check in future when I get the handset, the handset is not mine, it belongs to my sister.

**Reply by AnonymousWP** [reply]
https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/wiki/FAQ#where-are-the-logfiles

**Reply by SM-03** [reply]
> https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/wiki/FAQ#where-are-the-logfiles

@AnonymousWP 
Thanks. I found the log for vivo and am attaching it herewith. Please take a look and let me know. 
[UAD_20241015.log](https://github.com/user-attachments/files/17537945/UAD_20241015.log)

**Reply by AnonymousWP** [reply, marked as answer]
Hmm yeah, seems like Vivo is preventing you from disabling/uninstalling Vivo-made apps (or system apps in general):

```
[Recommended] pm disable-user --user 0 com.vivo.appstore -> 
Exception occurred while executing 'disable-user':
java.lang.SecurityException: Cannot disable com.vivo.appstore no root permission
```

I don't think there's any alternative at all then.🫤
